OrbitCore/Serialization.h

```diff
@@ -162,4 +162,20 @@ struct is_input_archive
                             cereal::traits::detail::decay_archive<Archive>>> {};
 template <typename Archive>
 inline constexpr bool is_input_archive_v = is_input_archive<Archive>::value;
-}  // namespace OrbitCore
\ No newline at end of file
+}  // namespace OrbitCore
+
+#ifdef ORBIT_FUZZING
+namespace cereal {
+// This overload takes precedence over the one provided by cereal, so it allows
+// us to change the behaviour. Of course, that's a (temporary) hack and will be
+// removed as soon as we remove cereal from the codebase. It only applies to the
+// fuzz testing build.
+template <class T>
+void CEREAL_SERIALIZE_FUNCTION_NAME(BinaryInputArchive& ar, SizeTag<T>& t) {
+  ar(t.size);
+  if (t.size > 100 * 1024 * 1024) {  // 100 MiB
+    throw Exception("size limit reached!");
```

Comment:
maybe use runtime_error as well?

Reply:
This is a `cereal::Exception` which inherits from std::runtime_error. I felt this was more appropriate. What do you think?

Reply:
sg